uint8 action
uint8 ADD = 1
uint8 REPLACE = 2

Bezier[] trajectory